### DISCUSSION TITLE
Poster creation from printed letter sized images. Edited

### DISCUSSION THREAD

**Discussion by Dajax55** [ImageMagick/ImageMagick#5011] [Help]
I using Win 10 and ImageMagick 7.1.0-28.  The command line so far creates the basic UN-resized printable sections so as to cut and reattaches to a larger poster.   Image overlap is about .5 inch and annotated for easier placement. (Which needs refining) .  Uploaded all files to www.runepic.com/poster which includes all files created.
EDIT:
Worked out a solution using an example below.

![capture_world](https://user-images.githubusercontent.com/102927309/162086321-7f5867d2-83e8-413f-9a39-c62bf73d4640.png)
Origional size 3918 x 3435 px.

using 
`magick convert  world.png -crop 6x4+96+96@  +repage  +adjoin -scene 01 -pointsize 34 ^
 -fill white -undercolor black -gravity southeast  -annotate 0  "%%p"   map_%%02d.png
 
magick montage -geometry 130x192+3+3 -label %%t  -tile 6x map_*.png map.html

:: Reference learning starting from  ...
:: convert rose: -crop 2x2+20+20@  +repage  +adjoin  rose_2x2+20+20@_%d.gif
:: https://legacy.imagemagick.org/Usage/crop/#crop_equal`

This creates needed pages to print and recombine.  Also an image map for reference.
 
![capture_thumbs](https://user-images.githubusercontent.com/102927309/162086978-5c2d08d3-27f0-47e1-8051-69c243f86c27.png)
The red line shows over lap.  map_02 over 01 and map_07 over 01.
Black line are the extended upper left pixel from annotate "box" for reference.

How to  annotate using 2 digit number starting with 1 like -scene for file name?

Thank you

**Comment by snibgo** [comment]
If you want the scene number instead of the index number of the index, use %s instead of %p. See the documentation https://imagemagick.org/script/escape.php.

There is no simple way to give the annotation number leading zeros.

**Reply by Dajax55** [reply]
Thank You for pointing me in right direction.
So -scene can use the setting to set the starting number for the current image sequence.
Written Scene Numbers - https://legacy.imagemagick.org/Usage/files/
 but annotate has no index or scene. Well not yet 👍 .

again tyvm
